com.github.ilai_deutel.kibi.yml

```diff
@@ -0,0 +1,34 @@
+app-id: com.github.ilai_deutel.kibi
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: kibi
+finish-args:
+  - --filesystem=host
+  - --filesystem=/tmp
+  - --filesystem=/var/tmp
```

Comment:
it doesn't display any GUI?

Reply:
Correct, it is a terminal-only text editor